grafana-plugin/pkg/plugin/install.go

```diff
@@ -104,33 +83,66 @@ func (a *App) handleInstall(w http.ResponseWriter, req *http.Request) {
 			}
 		}
 
-		w.Header().Add("Content-Type", "application/json")
-		if err := json.NewEncoder(w).Encode(installError); err != nil {
-			http.Error(w, "Failed to encode response", http.StatusInternalServerError)
-			return
-		}
-		w.WriteHeader(http.StatusBadRequest)
```

Comment:
this bit was moved into `handleInstall` handler below (to allow this method to be a bit more generic and be consumed ([here](https://github.com/grafana/irm/pull/69/files#diff-113878fd6f8c6e75911f61bb2c31d3833f905e7cbda9f00d5b795b443e2f200eR32-R46)) by the IRM backend plugin